### ISSUE TITLE
Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.s:nth-child(4))

### ISSUE THREAD

**Issue by Govardhani-T** [microsoft/onnxruntime#21294]
### GitHub Tags:
#A11ySev2; #A11yTCS; #DesktopWeb; #SH_ONNX Runtime & Ecosystem_Web_Mar2024; #ONNX Runtime & Ecosystem; #WCP; #Win11; #FTP; #P3_WCP; #ONNX Runtime; #ChromiumEdge; A11yWCAG2.1; #A11yMAS; #color-contrast; #WCAG1.4.3; #A11yAuto;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**

### Environment Details:
Application Name: WCP URLs - ONNX Runtime
URL: https://onnxruntime.ai/
Edge version: 123.0.2420.53 (Official build) (64-bit)
OS: Windows 11 version 23H2
### Steps to Reproduce

1. Hit the URL "https://onnxruntime.ai/docs/execution-providers/ROCm-ExecutionProvider.html".
2. "ONNX Runtime" AMD ROC-m page appeared.
3. Run Accessibility Insights for web.
4. Now check that Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds
### Issue: 
Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)
### Target application: 
AMD - ROCm | onnxruntime - https://onnxruntime.ai/docs/execution-providers/ROCm-ExecutionProvider.html
### Element path: 
.s:nth-child(4)
### Snippet: 
<span class="s">"ROCMExecutionProvider"</span>
### Related paths: 
  .language-python.highlighter-rouge:nth-child(27) > .highlight > pre
### How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 2.99 (foreground color: #50a04f, background color: #f5f6fa, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 126.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.43.0 (axe-core 4.8.4), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
### Attachment:
![image](https://github.com/microsoft/onnxruntime/assets/172380684/c2825a64-8161-4c99-a9ab-3db4de0c523d)

**Comment by Govardhani-T** [comment]
Verified the issue in the environment. issue is not fixed.
![image](https://github.com/user-attachments/assets/f0ddeb41-a59c-4732-9e73-5a8917847c9d)

**Comment by v-jagansai** [comment]
Verified the issue in the environment.: https://onnxruntime.ai/docs/execution-providers/ROCm-ExecutionProvider.html
issue is not fixed.PFA
![Image](https://github.com/user-attachments/assets/e6545486-b330-4794-a20a-7af434073db9)

**Comment by Govardhani-T** [comment]
Closing the bug as issue not reproing.![Image](https://github.com/user-attachments/assets/06473eb1-3071-490d-8c74-4707ca2972a0)

**Comment by PadminiGariki** [comment]
#Closed; #A11yFixed; #Regressed:10-07-25;
